.evergreen/config.yml

```diff
@@ -79,9 +79,6 @@ buildvariants:
     run_on:
       - ubuntu2204-arm64-small
     expansions:
-      # Needed to skip installation of the legacy shell in boostrap mongo-orchestration, which is
-      # not available on Ubuntu 22.04.
-      SKIP_LEGACY_SHELL: true
```

Comment:
This variable is no longer respected by drivers-evergreen-tools so I removed our uses of it